README.md

```diff
@@ -0,0 +1,50 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List
+
+"To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+
+## Built With
+
+- Major languages: HTML, CSS, JS. 
+- Technologies used: Visual Studio Code, Git/GitHub, Webpack.
+
+## Live Demo
+
+[[Live Demo Link](https://giuseppetg.github.io/To-do-list)]
```

Comment:
- [ ] [OPTIONAL] Kindly note that this is just the beginning of this project and it is not yet a requirement to deploy the project so let us remove this broken deployment link.